src/librustc/session/config.rs

```diff
@@ -558,9 +584,10 @@ options! {CodegenOptions, CodegenSetter, basic_codegen_options,
         "explicitly enable the cfg(debug_assertions) directive"),
     inline_threshold: Option<usize> = (None, parse_opt_uint,
         "set the inlining threshold for"),
+    sanitize: Option<Sanitize> = (None, parse_sanitize,
+        "choose the sanitizer to use"),
```

Comment:
For now, I think it's fair to say that this is a relatively  unstable option, so could this be behind the `-Z` flags instead of `-C`? Once we get more experience with it we may be able to promote it to `-C`